Makefile

```diff
@@ -0,0 +1,17 @@
+lint:
```

Comment:
You can have separate files one for backend  directory and one for frontend directory
ignore frontend makefile